config/application.rb

```diff
@@ -66,6 +66,18 @@ class Application < Rails::Application
 
     config.active_job.queue_adapter = :delayed_job
 
+    config.conference = {
+      events_per_page: 3,
+      default_logo:    'snapcon_logo.png',
+      default_color:   '#0B3559'
+    }
+
+    config.mailbot = {
+      bcc_address:    'messages@snap.berkeley.edu',
+      email_template: 'email_template',
```

Comment:
This, IMO should be just left in the mailbot file. It's unlikely to change without significant changes to app code, so it probably doesn't make sense as a config value.